homeassistant/components/vizio/strings.json

```diff
@@ -3,12 +3,12 @@
     "step": {
       "user": {
         "title": "Setup VIZIO SmartCast Device",
-        "description": "An Access Token is only needed for TVs. If you are configuring a TV and do not have an Access Token yet, leave it blank to go through a pairing process.",
+        "description": "An Access Token is only needed for TVs. If you are configuring a TV and do not have an [%key:common::config_flow::data::access_token%] yet, leave it blank to go through a pairing process.",
```

Comment:
Did you verify that this works ?

Reply:
So I just tested this, and locally it only replaces the token once. That means that while I have three references to the token, only one of them resolves (the first one). I can't tell whether this is specific to local development or a lokalise limitation

Reply:
To be clear, it did successfully replace the token within the string, but again it was just the first reference, including references in other key/value pairs

Reply:
it worked the second time I tested. Had to blow away my `venv` and start from scratch and it was a pretty old setup

Reply:
OK so did some further testing, and I was mistaken. When I ran `script/setup` everything looked good but it was probably because of old translations. When I ran `python3 -m script.translations develop` and then ran HA, I can see the token references instead of the translated text. Again, not sure if this is a Lokalise problem or a setup problem

Reply:
I don't think the develop module will resolve token references in looking at the code. I think this is specific to local dev. Not sure how to test this without a rework of the develop module or pushing this to `master` and seeing what happens.